### ISSUE TITLE
A11y: Ensures buttons have discernible text

### ISSUE THREAD

**Issue by heybran** [heybran/breeze-components#13]
`button` element needs a `aria-hidden=true` when `cc-side-nav` component does not have toggle behavior.

```
Element Location: cc-header,cc-side-nav,button[aria-controls="children"]
```
```
<button part="label" aria-controls="children">
  <slot name="label"></slot>
  <span part="toggle-icon" aria-hidden="true">
    <cc-icon icon="chevron-down"></cc-icon>
  </span>
</button>
```
